resources/references/upgrades/core/translation-handling.md

```diff
@@ -0,0 +1,62 @@
+---
+nav:
+  title: Migration Guide to internal Translation Handling
+  position: 10
+---
+
+# Migration Guide: Language Pack Plugin → Integrated Translation Handling
+
+Starting with Shopware **6.7.3**, translations are managed directly in Shopware. From **6.8**, the [Language Pack Plugin](https://store.shopware.com/en/swag338126230916f/shopware-language-pack.html) will no longer be compatible. Follow this guide to migrate safely.
+
+## What Changes
+
+- **From Shopware 6.7.3 onward**
+
+  - Translations can be installed via Shopware itself, the [Language Pack Plugin](https://store.shopware.com/en/swag338126230916f/shopware-language-pack.html) is not required to fetch the newest [Crowdin](https://crowdin.com/project/shopware6) translations.
+  - New CLI command available:
```

Comment:
You probably should add:
- Languages now have an active flag.
- Languages installed/managed from other sources do not need to register their locales in the admin anymore.

Reply:
done